### PR TITLE
Update docs to import graphql

### PR DESCRIPTION
**Pull request by tryzniak**
This is for #6914 .

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview for *using-postcss-sass* failed.

Built with commit 628e36e487e8bb75c6ff523a0c5ef539d30ffb80

https://app.netlify.com/sites/using-postcss-sass/deploys/5b61814a1f12b7103a7ef2fd

File: docs/docs/migrating-from-v1-to-v2.md [hunk 1]
```diff
@@ -316,6 +316,9 @@ The `sizes` and `resolutions` queries are deprecated in v2. These queries have b
 Update image query and fragment names:
 
 ```diff
+import React from "react"
++ import { graphql } from "gatsby"
```
Comment (pieh) [inline docs/docs/migrating-from-v1-to-v2.md]: I think this line should just be there but not marked as addition - importing graphql was covered in one of earlier points

Comment (tryzniak) [reply docs/docs/migrating-from-v1-to-v2.md]: Yeah, I was hesitant to add those. I will remove them!

**Comment by KyleAMathews** [comment]
Deploy preview for *gatsbyjs* failed.

Built with commit 994b39881618a478334106aeef094165ed22d319

https://app.netlify.com/sites/gatsbyjs/deploys/5b618512b312746adaa3f17c

**Pull request merged**